src/svm/eventsClient.ts

```diff
@@ -116,12 +126,12 @@ export class SvmSpokeEventsClient {
 
     // Fetch events for all signatures in parallel.
     const eventsWithSlots = await Promise.all(
-      filteredSignatures.map(async (signatureTransaction) => {
-        const events = await this.readEventsFromSignature(signatureTransaction.signature, program, anchorIdl, finality);
+      filteredSignatures.flatMap(async (signatureTransaction) => {
```

Comment:
`flatMap` does not seem to work as expected in async function and we anyways return `eventsWithSlots.flat()`, so I think we can leave just `map` here.

Reply:
Makes sense 👍
Updated.